File: test/cs/case100_class_instance.cs
using System.Collections.Generic;
using System.IO;
using System;
using TList = System.Collections.Generic.List;

public static class Program
{
    public class Box100
    {
        public int seed;

        public Box100(int seed)
        {
            this.seed = seed;
        }
        public int next()
        {
            this.seed = (this.seed + 1);
            return this.seed;
        }
    }

    public static void Main(string[] args)
    {
        Box100 b = new Box100(3);
        Console.WriteLine(b.next());
    }
}

File: test/py/case100_class_instance.py
# このファイルは `test/py/case100_class_instance.py` のテスト/実装コードです。
# 役割が分かりやすいように、読み手向けの説明コメントを付与しています。
# 変更時は、既存仕様との整合性とテスト結果を必ず確認してください。

import pathlib
from typing import List as TList


class Box100:
    def __init__(self, seed: int) -> None:
        self.seed = seed

    def next(self) -> int:
        self.seed += 1
        return self.seed


if __name__ == "__main__":
    b: Box100 = Box100(3)
    print(b.next())
